datadriven/examplesPipeline/geometryAwareSparseGrid.json

```diff
@@ -0,0 +1,38 @@
+{
+  	"dataSource": {
+    		"filePath": "/home/jamal/Documents/SGpp/datasets/mnist/mnist_[2]_28x28.train.arff",
```

Comment:
Why did you change that to an absolute path?

Reply:
my mistake I will fix it asap